### PR TITLE
Fixed some bugs

### PR DESCRIPTION
**Pull request by chaekmh**

### CODE REVIEW

**Review by whyWhyDev — APPROVED** [review]
Quick bug fix

**Review by nickstillman — APPROVED** [review]
lg2m

File: models/init.ts [hunk 1]
```diff
@@ -14,18 +14,18 @@ const envFileText = `
 dorm_databaseURL='postgresql://USERNAME:PASSWORD@localhost:5432/DATABASENAME?schema=public'
 
 # Please run the following command in your terminal to generate model : 
-#    deno run -A --unstable https://deno.land/x/dORM/models/generate.ts
+#    deno run --allow-read --allow-write --allow-net --unstable https://deno.land/x/dorm/models/model-generator.ts
```
Comment (m-aung) [inline models/init.ts:17]: great comment for developers who is using our dorm

**Review by m-aung — APPROVED** [review]
LGTM! good job Han

**Pull request merged**